src/dbnode/client/fetch_tagged_results_accumulator_merge_test.go

```diff
@@ -592,7 +592,7 @@ func (td testDatapoints) toRPCSegments(th testFetchTaggedHelper, start time.Time
 	for _, dp := range td {
 		require.NoError(th.t, enc.Encode(dp, testFetchTaggedTimeUnit, nil), fmt.Sprintf("%+v", dp))
 	}
-	reader, ok := enc.Stream(context.NewContext())
+	reader, ok := enc.Stream(context.NewBackground())
```

Comment:
This is a mechanical refactoring that I scoped to test files only

```
git grep --name-only "\.NewContext()"  | grep "_test.go" | xargs sed -i '' 's/\.NewContext()/\.NewBackground()/'
```

It was getting painful to update individual failing tests that were missing a go context when querying